rules/falco_rules.yaml

```diff
@@ -3398,6 +3398,35 @@
   priority: WARNING
   tags: [host, container, mitre_execution, mitre_defense_evasion, T1036.005, T1059.004]
 
+#The given rule is designed to detect suspicious activities related to disabling system recovery features. 
+#The rule looks for specific commands being executed such as rm -rf, chattr +i, mkfs.ext4 -E nodiscard, mount remount,ro, 
+#and disabling certain services such as systemd-backlight@.service, apport.service, rescue.service, emergency.service, and recovery-mode.service. 
+#https://attack.mitre.org/techniques/T1490/
+#Mitre Impact: Inhibit system recovery
+
+- list: recovery_files
+  items: [/etc/default/grub, /etc/inittab, /usr/lib/systemd/system/rescue.service]
+
+- rule: Disable Recovery Features
+  desc: Detects disabling system recovery features by deleting or disabling services and commands.
+  condition: >
+      (spawned_process) and 
+      ((open_write and fd.name in (recovery_files)) or
+      (proc.name = "rm" and proc.args contains "-rf") or
```

Comment:
I'm always a little bit concerned when I see rules matching on proc.args. For instance, `rm -rf` operation can be accomplished also with `rm -r -f` or `rm -fr`. Still didn't find a way to solve this, apart from exhaustively write down all cases (doable with two flags, but imagine with 3 or 4..)

Reply:
@loresuso  Thanks for the comments. will make changes accordingly.

Reply:
@pandyamarut - Keep us posted, when the changes are ready.

Reply:
@nidhisysdig pushed one commit.

Reply:
Ok for the remove part. Now the concern is that we are using different event types, so we have to not create impossible conditions. What I would like to check is:
- the part that checks `spawned_process` must be paired with the `proc.name` parts of the condition
- the parts with open_write or remove must be ORed together